quic/fizz/server/handshake/FizzServerHandshake.cpp

```diff
@@ -21,16 +23,16 @@ FizzServerHandshake::FizzServerHandshake(
     : ServerHandshake(conn), fizzContext_(std::move(fizzContext)) {}
 
 void FizzServerHandshake::initializeImpl(
-    std::shared_ptr<const fizz::server::FizzServerContext> context,
     HandshakeCallback* callback,
     std::unique_ptr<fizz::server::AppTokenValidator> validator) {
-  auto ctx = std::make_shared<fizz::server::FizzServerContext>(*context);
-  ctx->setFactory(cryptoFactory_.getFizzFactory());
-  ctx->setSupportedCiphers({{fizz::CipherSuite::TLS_AES_128_GCM_SHA256}});
-  ctx->setVersionFallbackEnabled(false);
+  auto context = std::make_shared<fizz::server::FizzServerContext>(
```

Comment:
is there a reason why this cannot simply be `context_ = std::make_shared<blah>();`

Reply:
I remove the field later on in #167 so I'd have to revert back to a local variable anyway.